docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-113.adoc

```diff
@@ -0,0 +1,47 @@
+== MonkeyLearn API Key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|TBD
+
+|Checkov Check ID
+|CKV_SECRET_113
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+A MonkeyLearn API Key is essential for accessing the MonkeyLearn text analysis services, which provide powerful machine learning models for text data. The key is critical for authenticating requests to the API and ensuring that only authorized applications can utilize the service. Exposure of this key can lead to unauthorized access, misuse of the API, and potential data leaks. It is vital to manage this key securely and regularly review your security practices to prevent any unauthorized usage. For detailed security guidelines and best practices, refer to the latest https://monkeylearn.com/api/v3/#authentication[MonkeyLearn documentation and support] resources.
+
+=== Fix - Buildtime
+
+*MonkeyLearn*
+
+If your MonkeyLearn API Key is exposed, it's important to act promptly to revoke the compromised key and generate a new one to secure access to the API. Follow these steps to replace the exposed key and update all dependent applications:
```

Comment:
If your MonkeyLearn API key is exposed, it’s crucial to take immediate action to prevent any unauthorized use. Regenerate a new key through your MonkeyLearn account and replace the exposed key with the new key in all relevant applications. By regenerating a compromised MonkeyLearn API Key and updating your application integrations, you not only secure your access to MonkeyLearn AI services but also maintain the confidentiality and integrity of your data flows.